src/vscode-bicep/schemas/bicepconfig.schema.json

```diff
@@ -362,6 +362,16 @@
                     }
                   ]
                 },
+                "prefer-property-names-no-quotation": {
```

Comment:
suggest: prefer-unquoted-property-names

Go ahead and make a reasonable decision and get it checked in.  Thanks!